pkg/federate/kubefed/federator.go

```diff
@@ -1,17 +1,59 @@
 package kubefed
 
 import (
+	"flag"
+	"fmt"
+	"sync"
+	"time"
+
 	"github.com/submariner-io/admiral/pkg/federate"
+	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+)
+
+var (
+	kubeFedNamespace string
 )
 
+func init() {
+	flag.StringVar(&kubeFedNamespace, "kubefed-namespace", "kube-federation-system", "The namespace in which the KubeFed control plane is deployed.")
```

Comment:
I believe we should also introduce the flags for setting the federation credentials (optional) (see my later "New" comment)?

In such way it'd be standard across all projects.